app/models/distribution.rb

```diff
@@ -36,6 +36,8 @@ class Distribution < ApplicationRecord
 
   before_save :combine_distribution
 
+  enum state: { started: 0, scheduled: 1, complete: 2 }
```

Comment:
Nice :+1:

Reply:
This is a *really minor* thing based on my past experience. I have a feeling we may need to possibly add other states at a later date. It would be nice if the highest number represented "complete". 

```suggestion
  enum state: { start: 0, schedule: 1, complete: 10 }
```

Could you change `complete` to `10`? I have no idea how many statuses we'll need to add over time, but that should give us a little breathing room. Also -- if you change the states to present tense, we can just use those methods and don't need to create any new ones! :)